midp/gfx.js

```diff
@@ -597,11 +593,11 @@ var currentlyFocusedTextEditor;
         return this.transY;
     };
 
-    Native["javax/microedition/lcdui/Graphics.getMaxWidth.()I"] = function() {
+    Native["javax/microedition/lcdui/Graphics.getMaxWidth.()S"] = function() {
```

Comment:
Can you add a simple test to make sure we don't regress those? (not a gfx test, a testlet)